src/controllers/collateralController.js

```diff
@@ -0,0 +1,48 @@
+require('dotenv').config();
+const Web3 = require('web3');
+const Tx = require('ethereumjs-tx');
+import {
+  AssetCollateral,
+  GAS_PRICE,
+} from '../constants';
+
+const ADDRESS = process.env.ADDRESS;
+const ADDRESS_PRIVATE_KEY = Buffer.from(process.env.ADDRESS_PRIVATE_KEY, 'hex');
+
+const web3Collateral = new Web3(new Web3.providers.HttpProvider(`https://ropsten.infura.io/v3/${process.env.INFURA_API_KEY}`));
+
+export const lockEscrow = async (assetId, assetManager, escrow) => {
+  return new Promise(async (resolve, reject) => {
+    try{
+      var txnCount = await web3Collateral.eth.getTransactionCount(ADDRESS);
+
+      const assetCollateral = new web3Collateral.eth.Contract(
+        AssetCollateral.ABI,
+        AssetCollateral.ADDRESS
+      );
+
+      var data = await assetCollateral.methods.lockEscrow(assetId, assetManager, escrow).encodeABI();
+      let rawTx = {
+        nonce: web3Collateral.utils.toHex(txnCount),
+        gasPrice: GAS_PRICE,
+        gasLimit: web3Collateral.utils.toHex(140000),
+        to: AssetCollateral.ADDRESS,
+        data: data,
+      }
```

Comment:
I don't know whether you copied pasted this code, but using `var` and `let` seems a bit odd considering we are on `es6`. Furthermore, we are not modifying any of this data so might as well call it `const`

Reply:
I haven't typed the word `var` in a while (you got me now) so it must have been a copy paste to get the transactionCount.

Fixed fixed.